aya/src/bpf.rs

```diff
@@ -176,10 +176,6 @@ fn compute_kconfig_definition(features: &Features) -> HashMap<String, Vec<u8>> {
                 Some('y') => to_bytes(1).to_vec(),
                 Some('m') => to_bytes(2).to_vec(),
                 Some('"') => {
-                    if raw_value.len() > 2 || raw_value.ends_with('"') {
-                        continue;
-                    }
-
```

Comment:
Oh yeah I completely inverted that check here...
Could you fix it and keep it for sanity checks as malformed data is still possible?